jni/CMakeLists.txt

```diff
@@ -111,7 +111,11 @@ endif ()
 if (${CONFIG_FAISS} STREQUAL ON OR ${CONFIG_ALL} STREQUAL ON OR ${CONFIG_TEST} STREQUAL ON)
     set(BUILD_TESTING OFF)          # Avoid building faiss tests
     set(BLA_STATIC ON)              # Statically link BLAS
-    set(FAISS_OPT_LEVEL generic)    # Keep optimization level generic
+    if(WIN32 OR ${CMAKE_SYSTEM_PROCESSOR} MATCHES "aarch64" OR NOT ${ENABLE_SIMD})
```

Comment:
Why wrap some variables in ${} and not WIN32? Lets be consistent here. Also, can you confirm that WIN32 will refer to the var and not a constant?

Reply:
updated the variable. `WIN32` is a variable which is True on a Windows machine.

Reply:
Right but why doesnt it need to be wrapped in ${} like ${WIN32}?

Reply:
Not sure. I took this reference, tried and it worked
https://stackoverflow.com/a/17845462